### ISSUE TITLE
Apply channel scope when evaluating user variables during release creation

### ISSUE THREAD

**Issue by MJRichardson** [OctopusDeploy/Issues#2759]
It would be nice to be able to use channel-scoped user variable in release version templates.

e.g.

```
#{Octopus.Version.Channel.LastMajor}.#{Octopus.Version.Channel.LastMinor}.#{Octopus.Version.Channel.NextPatch}-#{ReleaseTag}
```

where the `ReleaseTag` variable has different values per channel.

I believe this means simply applying the channel scope in the `EvaluateNextVersionTemplate()` method of  `ReleaseTemplateAction` (but I could be wrong).

From https://secure.helpscout.net/conversation/248169978/9555/?folderId=571746

**Comment by zentron** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Apply channel scope when evaluating user variables during release creation](https://trello.com/c/E2WL83tw/30-apply-channel-scope-when-evaluating-user-variables-during-release-creation)

**Comment by matt-richardson** [comment]
Duplicate. Was fixed via https://github.com/OctopusDeploy/Issues/issues/2967.

The variables `#{Octopus.Version.Channel.LastMajor}` etc are available from version `3.5.9`.
